example/lib/03.change_language_theme/widgets/dropdown_selector.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+class DropdownSelector extends StatelessWidget {
+  final IconData icon;
+  final ValueChanged<String> onChanged;
+  final String value;
+  final Iterable<String> values;
+
+  const DropdownSelector({
+    required this.icon,
+    required this.onChanged,
+    required this.value,
+    required this.values,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return DropdownButton<String>(
+      value: value,
+      items: values.map((String value) {
+        return DropdownMenuItem<String>(
+          value: value,
+          child: Text(value, style: const TextStyle(color: Colors.white)),
+        );
+      }).toList(growable: false),
+      icon: Icon(icon, color: Colors.white),
+      onChanged: (value) { if (value != null) onChanged(value); },
+      dropdownColor: Colors.black87,
+    );
+  }
+}
```

Comment:
https://medium.com/@alexey.inkin/how-to-force-newline-at-end-of-files-and-why-you-should-do-it-fdf76d1d090e